test/Log/LogServiceTest.php

```diff
@@ -65,7 +65,7 @@ public function setUp()
             'workspacePath' => $tempFolder,
             'log' => [
                 'format' => self::DEFAULT_FORMAT,
-                'level' => 'WARN'
+                'priority' => 'WARN'
```

Comment:
Careful with this line. This is an existing parameter in the configuration files. Change probably makes sense, but I wouldn't do it yet, because we will probably want to make a bigger change here. So please keep the original parameter name (level) for now.

Reply:
okay will change it back to level